app/metrics.py

```diff
@@ -77,6 +80,14 @@ def get_full_current_metrics(w3, beacon, beacon_spec, partial_metrics) -> PoolMe
         full_metrics.beaconValidators,
         full_metrics.activeValidatorBalance,
     ) = beacon.get_balances(slot, validators_keys)
+
+    block_number = beacon.get_block_by_beacon_slot(slot)
```

Comment:
don't we need to run smth like `get_first_non_missed_slot` from the `oracle-v3` branch in case of a missed block?

Reply:
We would not want to make many changes to the current version right now. The current version does not know how to find the previous slot on the CL and collect the report on it, e mean this applies not only to EL, but also to CL blocks, so the current version can't build a report if a block in the ref slot is missed. 

This is an old design bug that will be fixed in version 3